pkg/volume/aws_ebs/aws_ebs.go

```diff
@@ -393,6 +393,10 @@ func (c *awsElasticBlockStoreUnmounter) TearDown() error {
 
 // Unmounts the bind mount
 func (c *awsElasticBlockStoreUnmounter) TearDownAt(dir string) error {
+	if _, err := os.Stat(dir); os.IsNotExist(err) {
```

Comment:
The behavior of most of the `TearDownAt(...)` method for most volumes is identical, and it is implemented correctly in the common `volume/util` file (See `UnmountPath(...)` in https://github.com/kubernetes/kubernetes/blob/master/pkg/volume/util/util.go#L72). So instead of having it duplicated (with possible bugs like this, check all the existing `TearDownAt(...)` implementations, if they are not doing anything special/custom, just call the common util `UnmountPath(...)` method instead.